shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -153,13 +169,20 @@ @implementation FlutterTextInputPlugin {
 }
 
 - (instancetype)initWithViewController:(FlutterViewController*)viewController {
-  self = [super init];
+  // The view needs a non-zero frame.
+  self = [super initWithFrame:NSMakeRect(0, 0, 1, 1)];
   if (self != nil) {
+    _flutterViewController = viewController;
     _channel = [FlutterMethodChannel methodChannelWithName:kTextInputChannel
                                            binaryMessenger:viewController.engine.binaryMessenger
                                                      codec:[FlutterJSONMethodCodec sharedInstance]];
     _shown = FALSE;
-    __weak FlutterTextInputPlugin* weakSelf = self;
+    // NSTextView does not support _weak reference, so we have to
+    // use __unsafe_unretained and manage the reference ourselves.
+    //
+    // Since we will nil the handler in dealloc. the weakSelf should
```

Comment:
Remove the stray `.` here please. Also, as above, use something clearer than "we" (e.g., you can just say "|dealloc| removes the handler, so the pointer should be valid if the handler is ever called.").